libs/smartsync.js

```diff
@@ -80,6 +80,12 @@
             smartstoreClient = cordova.require("com.salesforce.plugin.smartstore.client");
         }
 
+        var sdkinfo = cordova.require("com.salesforce.plugin.sdkinfo");
```

Comment:
We test/debug sample apps in browsers (and some external developers have also used smartsync.js in web apps). So the code should be safe outside a container.
Could you do the following instead:

```javascript
if (window.cordova && window.cordova.require("com.salesforce.plugin.sdkinfo")) {
    var sdkinfo = cordova.require("com.salesforce.plugin.sdkinfo");
    if (typeof(sdkinfo.registerAppFeature) === 'function') sdksdkinfo.registerAppFeature('SJ');
 }
```
```